src/visualization/visualization.cc

```diff
@@ -20,6 +20,10 @@
 */
 //========================================================================
 
+// Anand's Notes:
```

Comment:
Please remove from here, you can add it to navigation.cc and particle_filter.cc if you'd like